### ISSUE TITLE
Seed Picker Emergency Recovery Is Unclear 

### ISSUE THREAD

**Issue by MinimalStructure** [btcguide/btcguide.github.io#46]
It tool me a long time to learn how to sign a transaction using the wallet generated by seed picker.

Using the an Electrum wallet was an exercise in futility, and when I tried using an old trezor one, I wasn't able to use a hardware wallet. 
![Screenshot_20201111-175940_DuckDuckGo](https://user-images.githubusercontent.com/69333458/98886420-e7cab980-2448-11eb-9909-c35e4c323890.jpg)

It finally worked when I entered the seedpicker wallet as a Trezor in the device tab instead of "other. "
Since the seedpicker wallet is only supposed to be used in an emergency, I was still able to keep the multi sig wallet airgapped, until I signed with the emergency Trezor.

Trezor would not have zero knowledge of seed beforehand.
Anyway, I thought about doing a pull request. I want to amend the paper wallet with the Trezor recovery and perhaps explain the process

**Comment by mflaxman** [comment]
Thanks, a PR would be great!

The main issue is that I'm not happy with any of the 3rd device options for emergency recovery (and reusing a 2nd device exposes a quorum of your seeds to a single device/manufacturer), so I'm kind of kicking that can down the road:
* Specter-DIY is great, but they're not available for sale :(
* Trezor does work, but has [serious limitations](https://btcguide.github.io/known-issues/hardware/trezor).
* BitBox02 is not perfect (also doesn't use its SD card for signing so doesn't support an airgap), but at least is stateful. When they add support for Specter-Desktop I'm going to look at it in-depth.
* My airgap CLI wallet [multiwallet](https://github.com/buidl-bitcoin/buidl-python/) is great, but CLI is too hard for most people. I haven't yet released [my gui version](https://github.com/mflaxman/multiwallet/), which is better in every way except software wallets are still inherently problematic. Eternally quarantining is easier for most people vs CLI/tails so this may be viable.
* Specter-Desktop can upload your seed to your internet-connected bitcoin core node, but this is a bad practice that should be discouraged.
* BlueWallet is working on a multisig offering that is also inherently risky, but for normies a mobile phone is probably less insecure than a general purpose OS. Ditto Gordian/Fully Noded, but those products unfortunately make Electrum look easy-to-use :(

So basically, I'm looking for a third device I can recommend (even with limitations/warnings) before investing in re-writing this section. That's part of why I'm building the GUI wallet!

**Comment by MinimalStructure** [comment]
Got it. I like your airgap wallet. It looks promising. 

It's probably just best to wait for that. Trezor does kinda suck, but it would work in an emergency. I haven't tried restoring the wallet with a ColdCard yet, but I'm fairly certain that would work too.
